spec/rails_helper.rb

```diff
@@ -2,8 +2,10 @@
 ENV["RAILS_ENV"] ||= "test"
 require File.expand_path("../../config/environment", __FILE__)
 
-abort("The Rails environment is running in production mode!") if Rails.env.production?
-require "rspec/rails"
+abort("The Rails environment is running in production mode!")
+if Rails.env.production?
+  require "rspec/rails"
+end
```

Comment:
This doesn't make sense. We don't want to run rspec in production.
We are also setting the Rails env specifically on line 1.
The `if` block should be around the `abort` call

Reply:
Ok, I will remove this.